src/Blocks/BlockButton.h

```diff
@@ -67,7 +67,7 @@ class cBlockButtonHandler :
 			case BLOCK_FACE_XM: return 0x2;
 			case BLOCK_FACE_XP: return 0x1;
 			case BLOCK_FACE_YM: return 0x0;
-			default:
+			case BLOCK_FACE_NONE:
```

Comment:
Firstly, having a value that is not in the enum is undefined behavior, secondly, the enum is contiguous, so tolua's range checks should pick up invalid values, and thirdly, I am making adding an element to the enum without updating it a compile error.

So there should be no invalid values passed in.